internal/infrastructure/db/migrations/20221006130646_integration_features.sql

```diff
@@ -21,7 +21,22 @@ INSERT INTO integration_features (feature_key, display_name, elastic_property) V
     ('odometer', 'Odometer', 'odometer'),
     ('oil', 'Engine Oil Life', 'oil'),
     ('tires', 'Tires', 'tires.frontLeft'),
-    ('speed', 'Speed', 'speed');
+    ('speed', 'Speed', 'speed'),
+    ('location', 'Location', 'latitude'),
+    ('battery_capacity', 'Battery Capacity', 'batteryCapacity'),
+    ('charging', 'Charging Status', 'charging'),
+    ('range', 'Range', 'range'),
+    ('vin', 'VIN', 'vin'),
+    ('cell_tower', 'Cell Tower Info', 'cell.ip'),
+    ('engine_runtime', 'Engine Run Time', 'runTime'),
```

Comment:
It's odd to me that in the first slot we don't separate "run" and "time", and then in the other two slots we do.

This is a very minor comment.